docs/kueue/manifests.yaml

```diff
@@ -0,0 +1,12716 @@
+apiVersion: v1
```

Comment:
This is going to rot over time. Could you replace this with a script to install kueue?

Reply:
Or find a way to make the footsprint lower? Maybe a kustomize script that enables pod integration?

Reply:
Possibly... however this is the approach recommended by the Kueue documentation: https://kueue.sigs.k8s.io/docs/installation/#install-a-custom-configured-released-version

Reply:
So then we replace this manifest with the script then.

I think over time this manifest will go bad and no longer work. Once the accelerator memory work merges, we would need to update this. It may be easier to just point to a released kueue manifest.

Reply:
OK. I'll implement a finer-grained patch but at the end of the day either we pin the Kueue version as I did and this will work only for so long or we don't and this will work only for so long. ;)

Reply:
Yea we don't really need to keep this around for too long so I'd rather have the script.

Once we have this in openshift/openshift AI we should just be able to use their Kueue to test this.

Reply:
I replaced the full `manifests.yaml` file with the much shorter `kueue-manager-config.yaml` file and tweaked deployment instructions accordingly.